README.md

```diff
@@ -222,6 +222,11 @@ results" in the Publish JUnit test result report configuration.
 
 ## Change Log
 
+### Version 0.71 (21 Sep 2021)
+- Updated scripts to use a different method for checking the presence of the
+  new reports api. Later versions of VectorCAST will require this version of the
```

Comment:
s/api/API/g

Reply:
This conflicts with @nbelliot 's previous review

Reply:
It is currently wrong.

Reply:
Seriously, "api" in lowercase *is grammatically wrong*.